python/ray/_private/services.py

```diff
@@ -433,7 +433,11 @@ def wait_for_node(
             return
         else:
             time.sleep(0.1)
-    raise TimeoutError("Timed out while waiting for node to startup.")
+    raise TimeoutError(
+        f"Timed out after {timeout} seconds while waiting for node to startup. "
+        f"Did not find socket name {node_plasma_store_socket_name} in the list "
```

Comment:
Is it user facing error message or internal? If it is user facing, the error message contains too much impl detail I think. If the socket name doesn't exist -> I think this basically means raylet wasn't started